src/sentry/event_manager.py

```diff
@@ -1877,8 +1872,12 @@ def _create_group(project: Project, event: Event, **kwargs: Any) -> Group:
         else None
     )
 
-    # get severity score for use in alerting
     group_data = kwargs.pop("data", {})
+
+    # add sdk tag to metadata
+    group_data.setdefault("metadata", {})["sdk"] = normalized_sdk_tag_from_event(event)
```

Comment:
Q: is there a value in putting this behind a flag if we cannot use Flagr here for performance reasons?

Reply:
I'm not sure I understand the question. Are you talking about the check below, the `if features.has("projects:first-event-severity-calculation", event.project)`? If not, am I missing something obvious? How does flagr come into it?

Separatly, but also a comment I'd stick on this line, I wonder if we maybe should store the full SDK value in the group metadata (normalized but not collapsed), and then do the collapsing where we need to for metrics. We don't really know what other scenarios might need to use the data, and I feel like we're better off erring on the side of more info rather than less. WDYT?

Reply:
> I'm not sure I understand the question. Are you talking about the check below, the `if features.has("projects:first-event-severity-calculation", event.project)`?

@lobsterkatie yeah, although in this case it would be something like `if features.has("organizations:grouping-store-sdk-meta", event.project.organization)` 

And more importantly it would have to be defined with `FeatureHandlerStrategy.INTERNAL`, with which as I understand value of the flag cannot be changed dynamically. It would let us limit potential blast radius to only S4S, but would still require revert if anything goes wrong.

Reply:
> I wonder if we maybe should store the full SDK value in the group metadata (normalized but not collapsed), and then do the collapsing where we need to for metrics. We don't really know what other scenarios might need to use the data, and I feel like we're better off erring on the side of more info rather than less. 

@lobsterkatie sure, version might also be useful. I'll dig into what other data is there.

Reply:
Ah, I see. I didn't realize you were thinking about putting this behind a flag. IDK that there's any reason to, though, is there? The normalization is protected by a try-except, and there's nothing inherently dangerous about adding an entry to a JSON blob. 

As for version/other metadata, many of those things (especially version) are things which aren't going to be stable between events in the group. Also, depending on the SDK, there can be a fair amount of dubiously-useful data (which SDK integrations were activated when the event was captured, for example). Until we need any of the rest of the data, my gut says we should keep it simple and just store the name - I just think we should store it in either its raw form or in a normalized-but-non-collapsed form, rather than its normalized-and-collapsed-for-metrics form.

Reply:
@lobsterkatie I've restricted the SDK meta to take only 3 fields: `name`, `version`, and `integrations`. These are [well defined in our API](https://develop.sentry.dev/sdk/event-payloads/sdk/), and I think they ought to be useful. There's also `packages`, but I'm not quite sure if that'd be useful here.

Reply:
I really don't think we should add version, because we're only doing this when we create the group, so all that will tell us is the first version we saw, even if the group hangs around for months or years. Integrations also might change over time. (Plus, most of them are on by default and it's rare for someone to turn them off, so the majority of this data will be the same for every group.)

Regardless, I chatted with Brian and we agreed on taking a less is more approach for now, and just adding the name. If we end up needing the integrations data, it's easy enough to add it back in at the time. That said, I like your approach of adding both the name and the normalized name, so I think let's keep the `sdk` object and add both to it. That also lets us add other data back in easily.